npm/private/npm_translate_lock_generate.bzl

```diff
@@ -321,6 +321,8 @@ def npm_link_all_packages(name = "node_modules", imported_links = []):
         npm_auth_password = "%s",""" % _import.npm_auth_password) if _import.npm_auth_password else ""
         maybe_dev = ("""
         dev = True,""") if _import.dev else ""
+        maybe_replace_package = """
```

Comment:
Can we add some `(...)` around this like most of (but not all 😢) the other templates here? There are way too many quotes within quotes here 😅